backend/editor/github_functions.py

```diff
@@ -137,6 +137,8 @@ async def create_pr(self, description) -> PullRequest:
         ### Description
         {description}
 
+        ### Link to the project in Taxonomy Editor
+        {settings.frontend_url}/{self.taxonomy_name}/{self.branch_name}/entry
```

Comment:
nit: I would feel better if you did that and added a redirection from `/` to `/entry` for project routes
```suggestion
        {settings.frontend_url}/{self.taxonomy_name}/{self.branch_name}
```